lib/js/jsffi.nim

```diff
@@ -113,6 +124,8 @@ proc toJs*[T](val: T): JsObject {. importcpp: "(#)" .}
 
 template toJs*(s: string): JsObject = cstring(s).toJs
 
+template `$`*(o: JsObject): cstring = cast[cstring](o.toString())
```

Comment:
Where is ``toString`` defined? What does it return, and why is it safe to cast to a cstring?

Reply:
`o` is a JsObject. `toString` gets compiled with the magic of the dot syntax.

Reply:
This doesn't return ``string`` but ``cstring`` and so does not adhere to Nim's stdlib. Change its name to ``toJsString`` or similar.